lib/direncrypt/direncryption.py

```diff
@@ -104,56 +103,131 @@ def encrypt_all(self):
         The files are recursively searched for in the source directory.
         """
         register = {}
+        printit("Encrypting all directory '{}', please wait...", self.plaindir)
         with Inventory(self.database) as inv:
-            register = inv.read_register()
-            inv.update_last_timestamp()
+            register = inv.read_all_register()
+            # treat regular files first
             files = self.find_unencrypted_files(register)
             for plainfile, val in files.items():
                 if not val['is_new']:
                     # remove old file in secure directory
                     encfile = inv.read_line_from_register(plainfile)
                     FileOps.delete_file(self.securedir, encfile)
                 encryptedfile = self.generate_name()
-                self.encrypt(plainfile, encryptedfile, inv)
+                self.encrypt(plainfile, encryptedfile, inv, False)
```

Comment:
Just for readability, please set `is_link=False`.

Reply:
Removed` False` value because default value of `is_link` parameter is now `False`